tst/standard/attr.tst

```diff
@@ -2210,6 +2210,217 @@ gap> P := DigraphRemoveAllMultipleEdges(ReducedDigraph(OnDigraphs(D, proj[2])));
 gap> IsIsomorphicDigraph(H, P);
 true
 
+# DigraphMaximalMatching
+gap> D := RandomDigraph(100);;
+gap> M := DigraphMaximalMatching(D);; IsMaximalMatching(D, M);
```

Comment:
It's probably better to create the graph at random on your computer, and then use its string representation explicitly here. Tests with randomness like this are often a bad idea.